.gitignore

```diff
@@ -3,6 +3,9 @@
 *.code-workspace
 # RIOT folder
 RIOT
+# generator folder
+dist/tools/m4a-generator/dist
+dist/tools/m4a-generator/m4agen.egg-info/*
```

Comment:
These lines has to be ignored, because these directories are built by the command `pip install .`(in the directory `dist/tools/m4a-generator/`)